### DISCUSSION TITLE
2024 3Q <성장 수집가> 김영승의 OKR

### DISCUSSION THREAD

**Discussion by ZeVicTech** [Pseudo-Lab/growth-collector#12] [우리들의 성장 목표!]
Objective: 주니어 AI 엔지니어로서의 갖추기

Key Result 1: 1주일에 한번 논문 구현 및 리뷰 포스트 작성
Key Result 2: 1주일에 두번 딥러닝 관련 수학 포스트 작성(확률과 통계, 선형대수, 미적분)
Key Result 3: AI 기술을 적용한 간단한 게임 프로젝트 개발

**Comment by ZeVicTech** [comment]
기간: 2024/08/08~2024/08/21
1. DDPM 논문 정리(수식 증명 및 이해를 중심으로)해서 블로그에 올리기
2. 확률과 통계 3개의 주제에 대해서 정리해서 블로그에 올리기

**Reply by yunhee1** [reply]
파이팅입니다!!

**Reply by leh60245** [reply]
화이팅입니다!!!!💪💪💪💪

**Comment by ZeVicTech** [comment]
1. DDIM 논문 리뷰 포스팅
2. 확률과 통계 주제(MLE 등) 3가지 관련해서 포스팅

**Reply by ZeVicTech** [reply]
이번에 면접 일정이 잡히면서 면접 준비를 하느라.
논문 리뷰 포스팅 못했습니다...(반성)
확률과 통계 주제(MLE 등) 3가지 관련해서 포스팅..못했습니다..(반성)

**Reply by yunhee1** [reply]
2주간 고생 많으셨습니다-!

**Reply by seriousran** [reply]
2주간도 고생 많으셨습니다..! 좋은 자리 찾으실 수 있기를 바라겠습니다아!!

**Comment by ZeVicTech** [comment]
- 2024/09/05~2024/09/19
- DDIM 논문 정리, BiRefNet 논문 정리 블로그에 올리기
- 확률과 통계 주제(MLE 등) 2개 포스팅
- 1일 1 회사 지원

**Comment by ZeVicTech** [comment]
2024/10/03~2024/10/17

- 코딩테스트 준비(1일 1코테 문제 풀기)
- 인공지능 논문 관련 블로그 포스팅 2개 올리기

**Reply by ZeVicTech** [reply]
https://nicator.tistory.com/1

**Reply by yunhee1** [reply]
포스팅해주신 글이 매우 유익하네요!
